### DISCUSSION TITLE
The kits are available on Tindie!

### DISCUSSION THREAD

**Discussion by philibertc** [philibertc/micronova_controller#9] [Announcements]
Hello to all, 
I have recently designed a PCB for this project for a cleaner result and to take less risks: no more connection errors like on a breadboard.
Several kits are available [here](https://www.tindie.com/products/philibertc/pellet-heater-controller/)!

**_Currently out of stock. Back in stock soon!_**

**Comment by geek78** [comment]
Hello

Hello
I would like to buy a complete kit . Do you think you will offer it for sale again soon?
Thank you

**Reply by philibertc** [reply]
Hello,

Everything will be available again by February 22.

**Reply by amateur67** [reply]
Bonjour,
Je voudrais un kit complet, c'est possible?
Fabian

**Reply by philibertc** [reply]
Bonjour,
Oui, c'est disponible ici : https://www.tindie.com/products/philibertc/micronova-pellet-stove-controller/
Si vous voulez un wemos d1 mini, je peux en commander et l'option sera disponible dans quelques jours

**Reply by amateur67** [reply]
Bonjour,
Le WEMOS sera programmé?
Je ne sais pas comment il faut le programmer ?
Si c'est possible, je voudrais un kit prêt à l'emploi :)
Fabian

**Reply by amateur67** [reply]
je relance

**Reply by philibertc** [reply]
Bonjour,

Désolé j'avais zappé la notif.
Le WeMos n'est pas pré-programmé car la configuration diffère pour chacun (identifiants mqtt).

**Reply by amateur67** [reply]
vous pourrez me guider pour cette opération?
La programmation ce fait via un port USB sur un ordi ou il faut un kit comme pour les PIC ?

**Reply by philibertc** [reply]
Oui bien sûr 
La programmation se fait directement par USB

**Comment by amateur67** [comment]
ok, je vous commande un kit complet, livraison sur la Belgique
C'est possible de communiquer via un autre canal, plus confidentiel?
Fabian

**Reply by philibertc** [reply]
Bonjour,
Oui, vous pouvez m'envoyer un message privé depuis la page Tindie.

**Reply by amateur67** [reply]
Bonjour,
La nouvelle carte Micronova est installée et à l'aide de votre print j'ai déjà fait la communication.
Mais dans le code, je n'arrive pas à trouver le réglage du ventilateur/extracteur de fumée?
Et quelles sont les valeurs que je peux encoder?
Vous pouvez m'aider pour les codes et valeur?
Bonne journée
Faiban
